server/src/main/java/org/elasticsearch/search/aggregations/bucket/geogrid/GeoGridAggregator.java

```diff
@@ -96,48 +96,6 @@ public void collect(int doc, long bucket) throws IOException {
         };
     }
 
-    // private impl that stores a bucket ord. This allows for computing the aggregations lazily.
-    static class OrdinalBucket extends InternalGeoGridBucket {
```

Comment:
@talevy I went ahead and just removed this class, tweaked the code below, and added a `bucketOrd` to `InternalGeoGridBucket` (that's similar to how other aggs do it).  

If you want I can revert this/the rest of the geogrid changes and wait for your pass over the agg.  I was just already committed by time I realized what was going on here, and it was quicker to tweak this section that revert :)